schema/index/index.go

```diff
@@ -105,7 +105,7 @@ func (b *Builder) StorageKey(key string) *Builder {
 //	func (T) Indexes() []ent.Index {
 //
 //		// Partial index on name where the entity is not deleted.
-//		index.Field("name").
+//		index.Fields("name").
 //			Annotations(index.Annotation{
 //				WhereClause: "deleted_at is not null",
```

Comment:
Should change these too but I'm not sure what to replace them with.

Reply:
maybe `field.Annotation{}`?

Reply:
I'll comment a suggestion.